open-budget-il/upload.py

```diff
@@ -217,10 +239,13 @@ class ULBudgetLine(UploadKind):
     KIND = "bl"
     CLS = BudgetLine
     KEY_FIELDS = [ 'year', 'code' ]
+    # Yonatan: i'd like to stick with dictionaries as they are more easily
```

Comment:
I think the best solution here is to use a namedtuple:

https://docs.python.org/2/library/collections.html#collections.namedtuple

wdyt?